### ISSUE TITLE
how to control cpu,gpu resource.

### ISSUE THREAD

**Issue by pkingwsd** [SciSharp/TensorFlow.NET#179]
in emgu.tensorflow, i can use ConfigProto to control cpu and GPU.

like this 

TensorFlow.ConfigProto configProto = new TensorFlow.ConfigProto();
configProto.InterOpParallelismThreads = 4;
configProto.IntraOpParallelismThreads = 4;
configProto.DeviceCount.Add("CPU" , 4);

var configProtoStr = configProto.ToProtobuf();
var p = Marshal.AllocHGlobal(configProtoStr.Length);
Marshal.Copy(configProtoStr, 0, p, configProtoStr.Length);
var opt = new TFSessionOptions();
mainSession = new TFSession(MainGraph, opt); 

in Tf.Net , how to do ?

**Comment by Oceania2018** [comment]
Fixed in v0.4.1. Added the `ConfigProto`.

**Comment by Nucs** [comment]
Example:
``` c#
var opts = new SessionOptions();
opts.SetConfig(new ConfigProto { InterOpParallelismThreads = (int) num_cpu, IntraOpParallelismThreads = (int) num_cpu, AllowSoftPlacement = true});
return new Session(graph, opts);
```
